tests/events.store.test.ts

```diff
@@ -1,15 +1,17 @@
-'use strict';
 
-const {createStore} = require('../src/events.store');
+import { createStore } from '../src/events.store';
+import { IRedisStore, RedisStore } from '../src/redis.store';
+import { expect } from 'chai';
+import td from 'testdouble';
 
 describe('events.store', () => {
   describe('#addEvent()', () => {
     const now = Date.now();
-    let actualEvent;
+    let actualEvent: any;
 
     before((done) => {
-      const itemsStore = td.object(['nextIndex', 'addItem']);
-      const subject = createStore({itemsStore});
+      const itemsStore = td.object<IRedisStore>(null as any);// (['nextIndex', 'addItem']);
```

Comment:
I doubt that `null as any` should be required. If a parameter is absolutely required, maybe `[]`? Or maybe no argument at all.